content/community/operators-rules.md

```diff
@@ -91,7 +102,7 @@ We recommend that all public servers publish contact addresses through this mech
 
 ### How do I get channel membership?
 
-First, don't fret about it. Membership is generally not required to participate in the chat, it is only used as an aid for moderation purposes when necessary.
+Membership is automatically granted by authbot when the criteria are met.
 
 If you have alternative proof that you are a server operator or a member of the XMPP Standards Foundation, you may request membership manually from one of the channel admins. Please don't ask for membership in any other circumstances - it won't be given.
```

Comment:
Do we want to keep this?

("you may request membership manually from one of the channel admins")

Seems contradictory to "Participation rights are managed automatically"

Reply:
I'm not sure. Specifically I don't want a flood of manual verification requests, and I'm not sure what manual verification would look like anyway. The only valid case I can think of is if someone *does* publish a contact address via '157, but prefer to join the MUC from a different address (which I think is a valid use case).

I'm fine with continuing to not grant participation rights to people who don't publish contact addresses.

Reply:
I would like to remove it.

Otherwise it may create a lot of effort for the moderation team for exactly that reason ("what does manual verification look like").